src/bionics.cpp

```diff
@@ -67,10 +69,141 @@ const efftype_id effect_visuals( "visuals" );
 const efftype_id effect_weed_high( "weed_high" );
 
 namespace {
+typedef std::pair<body_part, size_t> cbm_pair;
+
 std::map<std::string, bionic_data> bionics;
 std::vector<std::string> faulty_bionics;
+
+// similar to content of body_part_name() but with extra entry 'Other'
+std::string bodyparts[] = { bp_asText[0], bp_asText[1], bp_asText[2], bp_asText[3],
+                            bp_asText[4], bp_asText[5], bp_asText[6], bp_asText[7],
+                            bp_asText[8], bp_asText[9], bp_asText[10], bp_asText[11],
+                            _( "Other" ), _( "All" )
+                          };
```

Comment:
Code duplication is not a nice thing. You could write a function which would take care of the additional entries.